VBSHWWCORE/VBSHWW.cc

```diff
@@ -1103,7 +1112,15 @@ void VBSHWW::initSRCutflow()
             {
 
                 // Read jet p4
-                const LV& jet_p4 = nt.Jet_p4()[ijet];
+                LV jet_p4 = nt.Jet_p4()[ijet];
+                // Apply up/down corrections
+                if (jecvar == 1 || jecvar == -1)
+                {
+                    jec_unc->setJetEta(jet_p4.eta());
+                    jec_unc->setJetPt(jet_p4.pt());
+                    float jec_err = abs(jec_unc->getUncertainty(jecvar == 1))*jecvar;
```

Comment:
Should this be abs(jecvar) == 1?

Reply:
Nevermind this is correct.
https://github.com/cmstas/NanoTools/blob/master/NanoCORE/Tools/jetcorr/SimpleJetCorrectionUncertainty.icc#L68-L71